docs/administration-guide/secure-database-connections-with-ssl-certificates.md

```diff
@@ -0,0 +1,63 @@
+## Securing database connections using an SSL certificate
+
+If you'd like to connect your Metabase Cloud or self-hosted instance to a database, you can secure the connection using Secure Socket Layer (SSL) encryption with a certificate.
+
+Why you'd want to do this:
+
+- You're using Metabase Cloud and want to create a secure connection to a data warehouse that accepts a JDBC connection (e.g., PostgreSQL).
+- You're self-hosting Metabase and want to create a secure connection to an application database hosted by an external provider (this situation is rare).
```

Comment:
I would say here: You're self-hosting Metabase and want to ensure the identity of Data Warehouse hosted by an external provider.

Reply:
or you can also use this to ensure you're using the most strict connection parameters when connecting to your application database